public/javascripts/data.js

```diff
@@ -344,9 +344,18 @@ var dataNS = odkmaker.namespace.load('odkmaker.data');
                 }
             }
 
+            // actions
+            // see https://getodk.github.io/xforms-spec/#actions
+            var eventaction = {
+                name: 'odk:setgeopoint',
+                attrs: {
+                    'ref': xpath + control.name
+                }
+            }
```

Comment:
;

Reply:
Added, coming in next commit

Reply:
Fixed by https://github.com/getodk/build/pull/241/commits/c05881d8ded6a46d04cc21dedf115e8a74290d0c